uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/apis/entitlement-api.js

```diff
@@ -0,0 +1,88 @@
+define([
+  'leave-absences/shared/modules/apis',
+  'common/services/api'
+], function (apis) {
+  'use strict';
+
+  apis.factory('EntitlementAPI', ['$log', 'api', function ($log, api) {
```

Comment:
API name is in  caps as it is in sync with model and instance naming convention, let me know if its still need to be changed